app/plugins.js

```diff
@@ -241,7 +241,7 @@ function install(fn) {
       posix: 'npm prune && npm install --production'
     };
     // determine the shell we're running in
-    const whichShell = shell.match(/fish/) ? 'fish' : 'posix';
+    const whichShell = cfgShell.match(/fish/) ? 'fish' : 'posix';
```

Comment:
for extra ⭐️  i would rename this variable to `shellType` to be more descriptive